### PR TITLE
Troubleshooting update

### PR DESCRIPTION
**Pull request by arjuhe**
Added install bind-utils as it is not part of CentOS8 Appstream.
    Changed dig to host as the output is easier to read.

### CODE REVIEW

File: docs/troubleshooting/assisted_installer_failing_to_fetch_iso.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-> $ dig mirror.openshift.com
-> $ dig @8.8.8.8 mirror.openshift.com
-> $ dig @8.8.4.4 mirror.openshift.com
+> level=fatal msg="Failed to upload boot files" func=main.main.func1 file="/go/src/github.com/openshift/origin/cmd/main.go:191" error="Failed uploading boot files for OCP version 4.8: Failed fetching from URL https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso: Get \"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso\": dial tcp: i/o timeout"
+
+
+To check if the DNS for your container is configured and can resolve `mirror.openshift.com`.
+Try the following:
+  1. Connect to the containers shell.
+      > $ podman run -it ${container_id} /bin/bash
```
Comment (nocturnalastro) [inline docs/troubleshooting/assisted_installer_failing_to_fetch_iso.md:11]: Is `$container_id` the id of the installer container? It need to state that or people are likely to get confused.

We can probably add a line to grab that and store it in a var to make it even more straight forward.

Comment (arjuhe) [reply docs/troubleshooting/assisted_installer_failing_to_fetch_iso.md:11]: I can expand it by adding the `podman ps` command and showing sample output.

File: docs/troubleshooting/assisted_installer_failing_to_fetch_iso.md [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
-> $ dig @8.8.8.8 mirror.openshift.com
-> $ dig @8.8.4.4 mirror.openshift.com
+> level=fatal msg="Failed to upload boot files" func=main.main.func1 file="/go/src/github.com/openshift/origin/cmd/main.go:191" error="Failed uploading boot files for OCP version 4.8: Failed fetching from URL https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso: Get \"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso\": dial tcp: i/o timeout"
+
+
+To check if the DNS for your container is configured and can resolve `mirror.openshift.com`.
+Try the following:
+  1. Get the id of the http_store
+      > $ podman ps -a -f name=http_store
+      > CONTAINER ID  IMAGE                                               COMMAND               CREATED     STATUS         PORTS                 NAMES
```
Comment (nocturnalastro) [inline docs/troubleshooting/assisted_installer_failing_to_fetch_iso.md:12]: This doesn't get displayed properly. Check how it gets rendered on here.

**Review by snecklifter — APPROVED** [review]
LGTM

**Review by nocturnalastro — APPROVED** [review]

**Review by nocturnalastro — APPROVED** [review]

**Review by crwr45 — APPROVED** [review]

**Pull request merged**